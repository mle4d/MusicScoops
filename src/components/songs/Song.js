import React from 'react';
import PropTypes from 'prop-types';
import { Link } from 'react-router-dom';


function Song({ song, artist }) {
  return (
    <>
      <Link to={`/lyrics/${artist}/${song}`}><h3>{songTitle}</h3></Link>
    </>
  );
}

Song.propTypes = {
  song: PropTypes.string.isRequired,
  artist: PropTypes.string.isRequired
};

export default Song;
