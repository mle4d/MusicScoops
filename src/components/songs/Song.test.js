import React from 'react';
import { shallow } from 'enzyme';
import Song from './Song';

describe('Song component', () => {
  it('renders Song', () => {
    const wrapper = shallow(<Song />);
    expect(wrapper).toMatchSnapshot();
  });
});
