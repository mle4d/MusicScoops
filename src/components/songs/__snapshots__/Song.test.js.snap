// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Song component renders Song 1`] = `
<Fragment>
  <Link
    to="/lyrics/undefined/undefined"
  >
    <h3 />
  </Link>
</Fragment>
`;
